projectmanagement/src/main/java/life/qbic/projectmanagement/application/ExperimentInformationService.java

```diff
@@ -100,9 +100,18 @@ public void deleteExperimentGroup(ExperimentId experimentId, long groupId) {
     experimentRepository.update(experiment);
   }
 
-
-  public record ExperimentalGroupDTO(Set<VariableLevel> levels, int sampleSize) {
-
+  /**
+   * DISCLAIMER! This will remove all existing experimental variables and all defined experimental
+   * groups in a give experiment!
+   *
+   * @param experimentId the experiment reference to delete the experimental variables from
+   * @since 1.0.0
+   */
+  public void deleteAllExperimentalVariables(ExperimentId experimentId) {
+    Experiment experiment = loadExperimentById(experimentId);
+    experiment.removeAllExperimentalGroups();
```

Comment:
I would say, that if there are sample registered for an experiment, we shall not delete. Though I think the ExperimentInfoService should not know about the SampleInfoService to prevent unwanted service coupling. Maybe we need another service directive for cross-service actions?